tuxemon/states/monster_info.py

```diff
@@ -381,9 +383,14 @@ def add_menu_items(
         # Helper: find which stat a taste affects
         def get_stat_for_taste(slug: str) -> str | None:
             taste = lookup_tastes.get(slug.lower())
-            if not taste or not taste.modifiers:
+            if not taste:
                 return None
```

Comment:
Everything looks good, but this original version was already spot-on:
```python
if not taste or not taste.modifiers:
    return None
```
Gotta check this, no modifiers means things break. Then set it ready.

Reply:
Perfect, thank you!